Makefile.am

```diff
@@ -338,6 +407,7 @@ janus_pp_rec_CFLAGS = \
 
 janus_pp_rec_LDADD = \
 	$(POST_PROCESSING_LIBS) \
+	libjanus.la \
```

Comment:
I have noticed that there are many dependencies in transports/plugins/post-processor on the functions located in janus.c. I create shared object now based on that file.